src/main/java/cz/cvut/kbss/termit/rest/TermController.java

```diff
@@ -85,31 +89,35 @@ private URI getVocabularyUri(Optional<String> namespace, String fragment) {
                                     @RequestParam(name = "withReferences", required = false) boolean withReferences,
                                     @RequestParam(name = "property", required = false,
                                                   defaultValue = "[]") Set<String> properties,
-                                    @RequestHeader(value = HttpHeaders.ACCEPT, required = false) String acceptType) {
+                                    @RequestHeader(value = HttpHeaders.ACCEPT, required = false,
+                                                   defaultValue = MediaType.ALL_VALUE) String acceptType) {
```

Comment:
not sure why u need ALL_VALUE

Reply:
It's a default, because without default value `acceptType` is null, which cause NPX.